target-gen/src/commands/test.rs

```diff
@@ -25,9 +25,25 @@ pub fn cmd_test(
     definition_export_path: &Path,
     test_start_sector_address: Option<u64>,
 ) -> Result<()> {
+    ensure_is_file(target_artifact)?;
+    ensure_is_file(template_path)?;
+
+    anyhow::ensure!(
```

Comment:
Can we make this ensure_is_file too? :)

Reply:
No, because it doesn't need to exist already. It will be created ad a new file in that case, so it's different from the other cases.